src/v/pandaproxy/error.cc

```diff
@@ -190,6 +191,17 @@ std::error_condition make_error_condition(std::error_code ec) {
             return rec::kafka_authentication_error;
         }
         return {}; // keep gcc happy
+    } else if (ec.category() == make_error_code(pec::empty_param).category()) {
+        switch (static_cast<pec>(ec.value())) {
+        case pec::empty_param:
```

Comment:
nit: `[[fallthrough]];`

Reply:
I tend to do that only for cases where there is a statement. I don't think the compiler will warn unless there is a statement, anyhow.

Reply:
hmm, i thought clang-tidy warned in this case, but indeed im not see it.